keyboards/mechlovin/zed1800/oreum/keymaps/via/config.h

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
this also needs a license header, and an include guard.

```suggestion
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later

#pragma once

```